test/parallel/test-repl-null.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+require('../common');
+const repl = require('repl');
+const assert = require('assert');
+
+var replserver = new repl.REPLServer();
+
+replserver._inTemplateLiteral = true;
```

Comment:
I remember why I did it, but I do not remember why I chose that over possible alternatives.

I did that because this is going to test that `null` inside a template string gets treated like an empty string. (See the code comment below.)

It's possible that I was being lazy and/or copying from another test. It may be better to open an actual template string by emiting a backtick in a line. Or it's possible that I tried that and ran into problems.